src/components/ServiceDetails/AboutService/ServiceOverview.js

```diff
@@ -1,14 +1,22 @@
 import React from "react";
 import { withStyles } from "@material-ui/styles";
+import parseHtml from "html-react-parser";
 
 import { useStyles } from "./styles";
 import Tags from "./Tags";
 
 const ServiceOverview = ({ classes, description, tags }) => {
+  const parseDescription = description => {
+    if (description.startsWith("<div>")) {
```

Comment:
approving for now to test this, however lets explore genric way to identify html texts